examples/_d2n-flake-parts/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
From this example alone, it is not obvious if defining `perSystem` is actually necessary to make things work or if it's just an optional thing for customization.
Maybe we could have two examples for flake parts. One minimal one, and another one that shows how to override the package/devShell.

Reply:
I removed that part of the example, for now. I think later we can make a more extended example covering everything, after things get a little bit more stable.